### ISSUE TITLE
Oreo icon shape

### ISSUE THREAD

**Issue by c-e-p-x-u-o** [mkulesh/microMathematics#72]
Please add support for icon shape
https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive.html
https://material.io/guidelines/style/icons.html

**Comment by mkulesh** [comment]
Hi @c-e-p-x-u-o 
sorry for a late response. I Implemented your proposal. Please download the latest autobuild and test how does it now look.

**Comment by c-e-p-x-u-o** [comment]
Unable to install. It just says "App not installed."

**Comment by mkulesh** [comment]
Please note: The autobuild https://github.com/mkulesh/microMathematics/raw/autobuild/autobuild/microMathematics-v2.17.2.apk is not signed, i.e is not compatible with app version installed from any market.
Just uninstall official version of the app before you install autobuild APK

**Comment by c-e-p-x-u-o** [comment]
Yes, sorry. I realized it just after the deleted comment was written (:

What do you think about this concept:
![round](https://user-images.githubusercontent.com/14617699/39329889-69ced5ca-49a8-11e8-9604-21d67c15c99f.png)![square](https://user-images.githubusercontent.com/14617699/39329890-69f103ac-49a8-11e8-8e5c-97bbe248dd4a.png)
Or the same but in reverse color: blue -> white, white -> blue

**Comment by mkulesh** [comment]
thanks for idea, I will try it

**Comment by c-e-p-x-u-o** [comment]
the white penta-star inside is smaller in my imagination than on my images.

**Comment by c-e-p-x-u-o** [comment]
I think there should be no edge:
![1](https://user-images.githubusercontent.com/14617699/39363114-ccc9c6b0-4a31-11e8-88c1-e85325c678c7.png)![2](https://user-images.githubusercontent.com/14617699/39363115-ccf24a2c-4a31-11e8-8fbf-7a1dac3eaa6d.png)

**Comment by mkulesh** [comment]
yes, you are right, for adaptive icon I shall use a square shape instead of round one. Now, I changed it like this:
![ic_launcher_adapt](https://user-images.githubusercontent.com/30557348/39364557-a0d92e86-4a2e-11e8-937a-74b02d54014a.png)
Please take a look. 
PS. Thanks for testing. I only have a pure Android without third-party launchers, and, therefore, can only test in a standard environment.

**Comment by c-e-p-x-u-o** [comment]
Nice! But the center of the star should be a bit lower. See red marks on the second row.  
![all](https://user-images.githubusercontent.com/14617699/39365801-de4eb1f8-4a3a-11e8-8926-4e1c97f8837a.png)

**Comment by c-e-p-x-u-o** [comment]
You mean you have a pure Android 8.x (oreo) and it's impossible to change icon shape on it?

I'm using LineageOS 15.1 with Trebuchet launcher.

**Comment by mkulesh** [comment]
Yes, I currently use Nokia 8 with pure Android 8.1 as a reference device. Here, I can not change the icon style. 

Thank you for such a detailed test. Now, I centered the star exactly on the center of a circle shape, see https://github.com/mkulesh/microMathematics/blob/master/images/launcher/source.svg
For a square shape, the star may look as not centered since it is non-symmetric, but in this case it is an optical effect.

**Comment by c-e-p-x-u-o** [comment]
Awesome!
